### ISSUE TITLE
ascii-art subject

### ISSUE THREAD

**Issue by aS33f** [01-edu/public#879]
ascii-art
---
In the standard txt banner file, the "g" character is incorrectly positioned. Instead of the current 2-7, it should span over 3-8 char lines.

**Comment by aS33f** [comment]
or maybe is this as should be?

**Comment by nik-don** [comment]
@MSilva95 I have a feeling he might be correct. 

![asciiartletterg](https://user-images.githubusercontent.com/93073558/141198419-140be27c-65d1-49f0-8658-867e791c11c0.jpg)

But I assume it would involve having to edit a lot of other files instead of only the banner.

**Comment by MSilva95** [comment]
You're right, I'm going to change, we thought it was the length and not the height. Thank you for the catch!
